react/components/InfoCard/index.js

```diff
@@ -122,20 +125,21 @@ const InfoCard = ({
     formatIOMessage({ id: mobileImageUrl, intl })
   )
 
-  const containerStyle = isFullModeStyle
+  const containerStyle = isFullModeStyle && !lazyLoad
     ? { backgroundImage: `url(${finalImageUrl})`, backgroundSize: 'cover' }
     : {}
 
   const containerClasses = classNames(
-    `${handles.infoCardContainer} items-center`,
+    `${handles.infoCardContainer} items-center relative`,
     {
       [`flex-ns ${flexOrderToken} bg-base ph2-ns pb2 justify-between`]: !isFullModeStyle,
       [`bg-center bb b--muted-4 flex ${justifyToken}`]: isFullModeStyle,
+      'lazysizes': lazyLoad,
```

Comment:
Should it check if method is lazySizes?

Reply:
good point, it should

Reply:
actually, this is all up to render-runtime, so it should be removed altogether